src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1944,7 +1944,7 @@ const RegMask* Matcher::predicate_reg_mask(void) {
 
 // Vector calling convention not yet implemented.
 bool Matcher::supports_vector_calling_convention(void) {
-  return EnableVectorSupport && UseVectorStubs;
+  return EnableVectorSupport;
```

Comment:
You might want to remove the use of `UseVectorStubs` in `Matcher::vector_return_value` at L1951.
```
assert(EnableVectorSupport && UseVectorStubs, "sanity");
```

Reply:
Good catch. Fixed.